libraries/nomad.rb

```diff
@@ -0,0 +1,61 @@
+class NomadCluster < Inspec.resource(1)
+  name 'nomad_cluster'
+  desc 'Test the configuration of a Nomad cluster'
+  example "
+    describe nomad_cluster('http://localhost:4646') do
+      it { should be_reachable }
+      it { should have_leader }
+      its('servers_count'){ should be >= 3 }
+      its('datacenter'){ should include 'dc1' }
```

Comment:
`datacenter` is not implemented yet, right?

Reply:
Indeed :+1: